examples/server.rs

```diff
@@ -27,18 +27,33 @@ async fn handle(socket: TcpStream) -> Result<(), Box<dyn Error + Send + Sync>> {
     let mut connection = server::handshake(socket).await?;
     println!("H2 connection bound");
 
-    while let Some(result) = connection.accept().await {
-        let (request, mut respond) = result?;
-        println!("GOT request: {:?}", request);
-        let response = http::Response::new(());
+    let (mut request, mut respond) = match connection.accept().await {
+        Some(result) => result?,
+        None => {
+            println!("Didn't get a request");
+            return Ok(());
+        }
+    };
+
+    tokio::spawn(async move {
+        let _ = futures_util::future::poll_fn(|cx| connection.poll_closed(cx)).await;
```

Comment:
I think this change will only allow a single request, right? I'd probably say we should change the example to just _always_ spawn a task to handle the request. Something like:

```rust
while let Some(result) = connection.accept().await {
    tokio::spawn(handle_request(result));
}
```

Reply:
That is true.

Reply:
I've made it use `tokio::spawn` instead.

Btw the window stuff seems weirdly handled:

```
<<<< recv 16384
<<<< recv 16384
<<<< recv 16384
<<<< recv 16383
<<<< recv 1
<<<< recv 16384
<<<< recv 16384
<<<< recv 1696
```